soh/Makefile

```diff
@@ -54,7 +56,7 @@ INC_DIRS := $(addprefix -I, \
 )
 
 LDDIRS := $(addprefix -L, \
-	../external \
```

Comment:
On my build wasn't able to find libGLEW.a in `/usr/local/lib`
Copied it into StormLib/build and the linking succeeded.

Is there a better way to find GLEW?

Reply:
Actually on Linux I'm not sure we even need glew. Usually it is possible to directly link to libOpenGL.

Reply:
Does that include RayLib?

Reply:
I have no idea how raylib is set up, but I assume it applies there as well.